src/utils.py

```diff
@@ -714,6 +714,25 @@ def apply_operator(op: GroundNSRTOrSTRIPSOperator,
     return new_atoms
 
 
+def get_successors_from_ground_ops(atoms: Set[GroundAtom],
+        ground_ops: Collection[_GroundSTRIPSOperator],
```

Comment:
The type of this should be `GroundNSRTOrSTRIPSOperator`, matching `get_applicable_operators` and `apply_operator`. Also, this will let us change the two uses of get_applicable_operators/apply_operator: one in planning.py and the other in grammar_search_invention_approach.py. They can both use this function instead (remember to use `unique=False` though!).

Reply:
the one in planning.py needs the `nsrt`, not just the successor atoms, so I'm leaving that one untouched.